packages/@expo/cli/src/start/doctor/typescript/TypeScriptProjectPrerequisite.ts

```diff
@@ -77,7 +77,6 @@ export class TypeScriptProjectPrerequisite extends ProjectPrerequisite {
           // the package and attempts to resolve the module in the same process.
           { file: 'typescript/package.json', pkg: 'typescript' },
           { file: '@types/react/package.json', pkg: '@types/react' },
-          { file: '@types/react-native/package.json', pkg: '@types/react-native' },
```

Comment:
note here i've removed the `@types/react-native` package check in versioned cli.